plugins/kubernetes-backend/api-report.md

```diff
@@ -62,6 +62,7 @@ export interface ClusterDetails {
   caData?: string | undefined;
   // (undocumented)
   caFile?: string | undefined;
+  customResourceProfile?: string;
```

Comment:
I'd expect that we'd want to be able to list multiple profiles here potentially, or is it a useful simplification to only allow one?